### ISSUE TITLE
FURB118: false positive for pytest fixture

### ISSUE THREAD

**Issue by spaceone** [astral-sh/ruff#19305]
### Summary

```python
@pytest.fixture(scope='session')
def foo(bar):
    return foo['key']
```
cannot be written as `FURB118`:

`foo = pytest.fixture(scope='session')(operator.itemgetter('key'))` as the first argument of a pytest-fixture must be a keyword argument.

**Comment by MichaReiser** [comment]
Thanks. Ruff should probably ignore any decorated function (we could allow some well known decorators but I'm not sure if even that is worth it)
